### ISSUE TITLE
Strategy for prioritising the next set of test cases

### ISSUE THREAD

**Issue by lolaodelola** [w3c/aria-at-app#711]
I'd like to discuss with the group about the next set of test cases for ARIA-AT, Bocoup have experience running research for test cases for browsers (namely Firefox), and it may be useful to employ a similar strategy. We'd want to get input from developers, AT vendors, browsers, etc.

**Comment by jscholes** [comment]
@lolaodelola Happy to discuss, but I was under the impression that we already had a strategy for this at least in the short term.  Namely, the utilisation of APG visitor numbers as a metric of pattern/example interest, which is the intent we've loosely been communicating to vendors.  We don't yet have those numbers in our test plan tracking spreadsheet, so we should talk about how we get them there.

**Comment by lolaodelola** [comment]
You're right @jscholes, this is referring to a possible stretch goal. The APG traffic data is what we'll definitely be using but I also wanted to get a feel from the group about possibly extending the sources to places like stack overflow, httparchive & doing a framework analysis.

**Comment by mcking65** [comment]
We have sufficient ability to prioritize over the next 6 to 12 months with APG data and real time feedback, and we are stretched thin getting the basic plumbing of the project functioning sufficiently well. I believe every bit of energy possible should be focused on core capabilities until they are solid.

Moving this issue to the aria-at repo.
